sos/collector/transports/oc.py

```diff
@@ -71,91 +68,67 @@ def get_node_pod_config(self):
                     {
                         "name": "host",
-                        "hostPath": {
-                            "path": "/",
-                            "type": "Directory"
-                        }
+                        "hostPath": {"path": "/", "type": "Directory"},
                     },
                     {
                         "name": "run",
-                        "hostPath": {
-                            "path": "/run",
-                            "type": "Directory"
-                        }
+                        "hostPath": {"path": "/run", "type": "Directory"},
                     },
                     {
                         "name": "varlog",
-                        "hostPath": {
-                            "path": "/var/log",
-                            "type": "Directory"
-                        }
+                        "hostPath": {"path": "/var/log", "type": "Directory"},
                     },
                     {
                         "name": "machine-id",
                         "hostPath": {
-                            "path": "/etc/machine-id",
-                            "type": "File"
-                        }
-                    }
+                            "path": "/etc/machine-id", "type": "File"
+                        },
+                    },
                 ],
                 "containers": [
                     {
                         "name": "sos-collector-tmp",
                         "image": "registry.redhat.io/rhel8/support-tools"
-                                if not self.opts.image else self.opts.image,
-                        "command": [
-                            "/bin/bash"
-                        ],
-                        "env": [
-                            {
-                                "name": "HOST",
-                                "value": "/host"
-                            }
-                        ],
+                        if not self.opts.image
+                        else self.opts.image,
+                        "command": ["/bin/bash"],
+                        "env": [{"name": "HOST", "value": "/host"}],
                         "resources": {},
                         "volumeMounts": [
-                            {
-                                "name": "host",
-                                "mountPath": "/host"
-                            },
-                            {
-                                "name": "run",
-                                "mountPath": "/run"
-                            },
-                            {
-                                "name": "varlog",
-                                "mountPath": "/var/log"
-                            },
+                            {"name": "host", "mountPath": "/host"},
+                            {"name": "run", "mountPath": "/run"},
+                            {"name": "varlog", "mountPath": "/var/log"},
```

Comment:
Same kinda thing here. I get that these dict entries/definitions can fit in a single line, but having one-line-per-key is more human-friendly, at least to me. Happy to hear dissenting opinions.

Reply:
Yep, the un-posted comment from last night mentioned this entire file (or any other file where non f-string change occurred) would be rolled back to its original format unless people wanted them to stay. I was fairly displeased to find  these were touched at all when reviewing the diffs, as they did not contain f-string changes at all. It appears as if a data structure had an fstring change to it, and within around 10 lines inside the data-structure the sourcery might adjust the format, it went ahead as if allowing f-string rules meant within +/-10 lines of the fstring adjustment other format changes were allowed.